content/source/docs/cloud/integrations/service-now/index.html.md

```diff
@@ -167,23 +163,22 @@ Class Name | Description
 `tf_vcs_record` | Manage ServiceNow Terraform VCS Repositories Table Records
 `tf_workspace` | Resources for Terraform Workspace API Requests
 
-### Example Service Catalog Workflows
+### Example Service Catalog Flows and Actions
 
-The ServiceNow Example Workflows can be found in the ServiceNow Studio > Workflow > Workflow. By default, the workflows execute upon submitting an order request for the various catalog items. Admins can modify the workflows to wait on an approval action, include approval rules, and specify approver groups.
+The ServiceNow Example Flows can be found in the ServiceNow Studio > Flows. By default, the Flows execute upon submitting an order request for the various catalog items. Admins can modify the Flows and Actions to wait on an approval action, include approval rules, and specify approver groups.
```

Comment:
Are Flows and Flows and Actions really proper nouns? Other places in this PR they are lower case.

Reply:
Fixed! (Hopefully 😅 )